### PR TITLE
[MB-8979] Add a  sidebar to the payment request tab

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description

This PR adds a sidebar to the left of the payment request details page. It's similar to other left side bars found on the Move details page

## Reviewer Notes

- I didn't create a more general sidebar component because each place in the app that displays a similar sidebar is already reusing the leftNav component. 
- It's been awhile since I've written front end test and I haven't used the react testing library yet --please double check that the test I'm adding here make since and follow conventions 
- Is it correct that the payment request tab in the sidebar shouldn't appear when there aren't any payment request?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate server_run client_run
```
- Navigate to officelocal, sign in as a TIO and select a payment request (ideally one that's long enough to have scrolling on the page).
-  I've used the PARAMS move to test: http://officelocal:3000/moves/PARAMS/payment-requests. 
- This work is also deployed here https://office-milmove-pr-7179.mymove.sandbox.truss.coffee/

Verify that
- Sidebar has been added to the http://officelocal:3000/moves/:moveCode/payment-requests page
- Sidebar takes the user to each section when the item is clicked 
- Sidebar highlights each item when scrolling through the page

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8979) for this change

## Screenshots

Screen Shot 2021-08-11 at 12 59 50 PM

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1624,7 +1624,7 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@fortawesome/fontawesome-common-types@^0.2.35", "@fortawesome/fontawesome-common-types@^0.2.36":
+"@fortawesome/fontawesome-common-types@^0.2.36":
```
Comment (YanZ777) [inline yarn.lock:1627]: Do you know why this yarn.lock is showing up?

Comment (pearl-truss) [reply yarn.lock:1627]: i'm not sure, I reset it to match master since I'm not intentionally adjusting any libraries

**Review by YanZ777 — COMMENTED** [review]
Your code looks fine, but there is definitely something wonky going on with the yarn.lock file.

**Review by cosburn — APPROVED** [review]
Looks good to me!

**Review by YanZ777 — APPROVED** [review]
Changes look good to me!

**Pull request merged**